packages/libtinfo.rb

```diff
@@ -18,8 +18,8 @@ class Libtinfo < Package
   def self.install
     # See https://forums.opensuse.org/showthread.php/446927-missing-library-libtinfo-so-5.
     # See also http://www.linuxforums.org/forum/installation/6251-libtinfo-so-5-a.html.
-    system "mkdir -p #{CREW_DEST_LIB_PREFIX}"
-    system "ln -sf #{CREW_LIB_PREFIX}/libncurses.so.6 #{CREW_DEST_LIB_PREFIX}/libtinfo.so.5"
-    system "ln -sf #{CREW_LIB_PREFIX}/libncurses.so.6 #{CREW_DEST_LIB_PREFIX}/libtinfo.so.6"
+    FileUtils.mkdir_p #{CREW_DEST_LIB_PREFIX}
+    FileUtils.ln_sf '#{CREW_LIB_PREFIX}/libncurses.so.6', '#{CREW_DEST_LIB_PREFIX}/libtinfo.so.5'
+    FileUtils.ln_sf '#{CREW_LIB_PREFIX}/libncurses.so.6', '#{CREW_DEST_LIB_PREFIX}/libtinfo.so.6'
```

Comment:
You will need to encapsulate with double instead of single quotes.  Also, no need to wrap the constant in `#{...}`.  `FileUtils.mkdir_p CREW_DEST_LIB_PREFIX` should work.  In the future, please test first before committing changes.

Reply:
Done